exercises/practice/space-age/.meta/src/reference/java/SpaceAge.java

```diff
@@ -65,9 +61,9 @@ private double getRelativeOrbitalPeriod() {
     }
 
     private double calculateAge(Planet planet) {
-        double age = seconds / (EARTH_ORBITAL_PERIOD_IN_SECONDS * planet.getRelativeOrbitalPeriod());
-
-        return new BigDecimal(age).setScale(PRECISION, BigDecimal.ROUND_HALF_UP).doubleValue();
+        double planetYears = (seconds / EARTH_ORBITAL_PERIOD_IN_SECONDS) / planet.getRelativeOrbitalPeriod();
+        
+        return (double) Math.round(planetYears * 100) / 100;
```

Comment:
This change is due to the fact that `BigDecimal.setScale(int, int)` is deprecated, correct?

I looked around and it seems like the recommended way is to use [`BigDecimal.setScale(int, RoundingMode)`](https://docs.oracle.com/javase/8/docs/api/java/math/BigDecimal.html#setScale-int-java.math.RoundingMode-). Although I don't mind this approach either.

Up to you to choose which way you prefer!

Reply:
I don't have a strong opinion on this one, so if you see the BigDecimal approach better tell me and I will make it work hahaha

Reply:
To be honest my opinion is that the exercise should have to round the results at all, instead the test cases should be implemented so that they compare the results only for the first two decimals...

Nothing in the instructions talks about rounding, I've always found it strange that this is something the students should figure out...

Reply:
Yeah I agree, we could perform the rounding or decimal check in the tests, should be taken as a separate issue or you want me to try solve it in this PR?

Reply:
Feel free to change it in this PR. I suggest having a look at the [`AbstractDoubleAssert.isEqualTo(double, Offset<Double>)`](https://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/AbstractDoubleAssert.html#isEqualTo(double,%20org.assertj.core.data.Offset)) method.

Reply:
Thanks for the advice! I will give it a try!

Reply:
Done!